src/plugins/gui/scenes/RewardsCardDashboardScene.tsx

```diff
@@ -116,9 +114,9 @@ const CardListItemContainer = styled(View)(props => ({
   marginRight: props.theme.rem(1)
 }))
 
-const BottomFloat = styled(View)<{ bottomSpace: number }>(props => ({
+const BottomFloat = styled(View)(props => ({
   alignSelf: 'center',
-  bottom: props.bottomSpace,
+  bottom: 0,
```

Comment:
Optional: Can be removed completely now that it's zero?

Reply:
Nah, cause that’s the way to anchor it to the bottom using absolute positioning

Reply:
@peachbits Exactly what @samholmes said. Otherwise the button ends up at the very top